RELEASE_NOTES.rst

```diff
@@ -7,6 +7,16 @@ All changes
 - Adjust the Westeros reporting tutorial to pyam 1.0 deprecations (:pull:`492`).
 
 
+.. _v3.4.0:
+
+v3.4.0 (2021-MM-DD)
+===================
+
+All changes
+-----------
+
+- Update `years_active()` to use sorted results (:pull:`491`).
```

Comment:
Please:
- Keep only this 1 line above/below the existing one, under the existing heading Next release / All changes.

  We don't put release numbers into the file until the time a release is made. The next release could be 3.3.1, 3.4.0, or 4.0.0; we don't know yet.
- Use the [`meth`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-meth) Sphinx role to link to the function:
  ```
  :meth:`.years_active`
  ```

Reply:
@khaeru changes made.